src/stories/components/custom-button/custom-button.tsx

```diff
@@ -95,69 +95,67 @@ export const CustomButton = ({
   width,
   type = 'button',
   padding = '15px 16px',
+  isLight,
   ...props
-}: CustomButtonProps) => {
-  const { isLight } = useThemeContext();
-  return (
-    <Container
-      padding={padding}
-      active={active}
+}: CustomButtonProps) => (
+  <Container
+    padding={padding}
+    active={active}
+    allowsHover={allowsHover}
+    className={`${props.className} no-select`}
+    isLight={isLight}
+    buttonType={buttonType}
+    type={type}
+    disabled={props.disabled}
+    onClick={props.onClick}
+    styles={{
+      backgroundColor: isLight
+        ? active
+          ? customStyles[buttonType].activeBackground
+          : customStyles[buttonType].background
+        : active
+        ? customStyles[buttonType].activeBackgroundDark
+        : customStyles[buttonType].backgroundDark,
+      borderColor: isLight
+        ? allowsHover
+          ? active
+            ? customStyles[buttonType].activeBorderColor
+            : customStyles[buttonType]?.borderColor
+          : customStyles[buttonType].borderColorMobile
+        : allowsHover
+        ? active
+          ? customStyles[buttonType].activeBorderColorDark
+          : customStyles[buttonType]?.borderColorDark
+        : customStyles[buttonType].borderColorMobileDark,
+      ...props.style,
```

Comment:
note that this added unwanted changes to the auditor comment form component. You can see the visual change on chromatic